GVRf/Framework/.cproject

```diff
@@ -5,14 +5,14 @@
 			<storageModule buildSystemId="org.eclipse.cdt.managedbuilder.core.configurationDataProvider" id="com.android.toolchain.gcc.1319470856" moduleId="org.eclipse.cdt.core.settings" name="Default">
 				<externalSettings/>
 				<extensions>
+					<extension id="org.eclipse.cdt.core.ELF" point="org.eclipse.cdt.core.BinaryParser"/>
```

Comment:
I don’t think so. I did not explicitly make this change myself but Eclipse might have done it.

From: Danke Xie [mailto:notifications@github.com]
Sent: Monday, April 11, 2016 11:25 AM
To: Samsung/GearVRf <GearVRf@noreply.github.com>
Cc: Nola Donato <nola.donato@samsung.com>
Subject: Re: [Samsung/GearVRf] Multilight pr (#491)


In GVRf/Framework/.cproject<https://github.com/Samsung/GearVRf/pull/491#discussion_r59255340>:

> @@ -5,14 +5,14 @@

>                      <storageModule buildSystemId="org.eclipse.cdt.managedbuilder.core.configurationDataProvider" id="com.android.toolchain.gcc.1319470856" moduleId="org.eclipse.cdt.core.settings" name="Default">

>                              <externalSettings/>

>                              <extensions>

> +                                   <extension id="org.eclipse.cdt.core.ELF" point="org.eclipse.cdt.core.BinaryParser"/>

Are the changes on this file necessary?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Samsung/GearVRf/pull/491/files/3d37246ccf99f0715f5ad966e0fb8da7980c04ec#r59255340>